### ISSUE TITLE
Starts to randomly download/sync files

### ISSUE THREAD

**Issue by chefexperte** [nextcloud/android#13977]
### ⚠️ Before posting ⚠️

- [x] This is a **bug**, not a question or an enhancement.
- [x] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [x] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

The thing is, I didn't do anything specific before this started happening.

### Expected behaviour

Nextcloud does not download random files without any option for me to stop it. 

### Actual behaviour

It does that anyways.
I just opened the app because nextcloud was spamming notifications that instantly disappeared again. When I opened the app I saw the sync icon (blue circle with arrow) on random folders, and saw the app was using lots of network data.
Sync had been activated for a lot of folders I haven't ever uploaded to or downloaded from within the app. Clicking cancel sync also did not stop it downloading stuff.
I can't find any way to stop Nextcloud from downloading tons of data I do not need on my phone.  

### Android version

15

### Device brand and model

Google Pixel 7

### Stock or custom OS?

Custom (explain in "additional information")

### Nextcloud android app version

3.30.4

### Nextcloud server version

29.0.8

### Using a reverse proxy?

Yes

### Additional information

I am using Graphene OS.
I don't have the beta or dev version.

**Comment by spaceChRiS** [comment]
I can confirm this, on my wife's motorola g54 it happened as well after auto-install of 3.30.4 over f-droid, 15 GB transfer over mobile network, but only one upload folder and only upload over wifi allowed.

**Comment by joshtrichards** [comment]
Duplicate of #13872

**Comment by spaceChRiS** [comment]
@joshtrichards, I am not sure if the issue is a duplicate (actually I am pretty sure it is not). First, it happened only after installing the version 3.30.4, which is mentioned under mitigation in #13872: “We strongly recommend that all users to upgrade to version 3.30.4 or later using your preferred method to resolve these issues.” Furthermore, after performing mitigation option 1 in #13872, it does not stop to do upload/download operations of old files.

**Comment by joshtrichards** [comment]
Does https://github.com/nextcloud/android/issues/13872#issuecomment-2461952811 apply to your situation maybe?

**Comment by spaceChRiS** [comment]
What made me ignore this was the sentence “still experiencing problems after upgrading to version 3.30.4”, as my wife did not have any problem before 3.30.4, and as auto-update was enabled, she likely had 3.30.2 or .3 installed without any issue. However, reading again carefully through the message, it may be the case that it is the same issue, and it's difficult to reverse-engineer which version exactly was installed before the auto-update.

And especially, we are facing the issue in https://github.com/nextcloud/android/issues/13872#issuecomment-2462165304, but maybe we did not wait long enough and the temporary cue is big. My wife has approx. 100 GB of data in her account (luckily I am not affected with my ~500 GB, as I installed back in the days when auto-update was not enabled in f-droid).

So many thanks for the support, and I'll report any new findings #13872.
